mmvstr.h

```diff
@@ -126,18 +126,26 @@ with 'let(&' and thus has the same effect as 'let(&'.
 
```

Comment:
I'd like to see a usage example for temp_vstring in the introducing comment block.  Examples should be updated to use vstringdef.